packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -848,7 +848,7 @@ describe('wp-cli command', () => {
 	 * We don't need the WordPress context for this test.
 	 */
 	test('wp-cli displays the version', async () => {
-		await executeWPCli(['cli', 'version']);
-		expect(output).toMatch(/WP-CLI (\d\.?)+/i);
+		const { stdout } = await executeWPCli(['cli', 'version']);
```

Comment:
The function `executeWPCli` previously output to console  - captured by the mock console in the tests - but the updated version from the fork in `Automattic/studio` now returns an object with `stdout` and `stderr`. The latter seems better in terms of flexibility, but perhaps it should be consistent with `executePHP` (which outputs to console).

Reply:
Using an object with `stdout` and `stderr` will give us more flexibility for the future 👍

Reply:
Done.